sips/networking/last_change_round.md

```diff
@@ -0,0 +1,99 @@
+| Author     | Title                                             | Category | Status |
+|------------|---------------------------------------------------|----------|--------|
+| Niv Muroch | Handle non committee change round for all signers | Core     | Draft  |
+
+**Summary**  
+Allow nodes to save non committee (validators that are not belong to the node) change round messages in order to serve
+other nodes when requesting last change round messages.
+
+Rational & Design Goals  
+When using subnets instead of topic per validator there is a need for supporting get last change round from any peer in
+the subnet. in order to supports that, each node (peer) needs to handle incoming change round messages and save it per
+signer and only the highest once. the reason for saving messages per signer is to keep the same behavior as it was with
+validator topic.
+
+In order to achieve change round quorum we need 3 different signers. other option is f+1 where we need 2 signers.
+save change round per signer instead of the last one solve the that needs.
+
+----------
+
+**Specification**
+
+**Networking**
+
+*Saving*
+Each operator peer listen for the non committee change round messages within it's subscribed subnets.
+once receive non committee change round message, the following process run -  
+
+- When local not exist. New message saves instantly.
+- New change round message should be with higher or equal height than the local message.
+  If higher, save instantly. if not, check if round is higher. if so, update.
+- need to maintenance the last change round message for each of the signers in the specific validator committee.
+
+```go
```

Comment:
If you need code changes better put it in the spec no? 
SIPs talk more about spec than implementation

Reply:
sync change round is not exist in spec. we can exclude this implementation explain from SIP. WDYT?